src/main/java/xyz/nucleoid/creator_tools/workspace/MapWorkspaceManager.java

```diff
@@ -216,4 +219,26 @@ private RuntimeWorldConfig createDefaultConfig() {
                 .setDimensionType(DimensionTypes.OVERWORLD)
                 .setGenerator(generator);
     }
+
+    /**
+     * Migrates the file storing map workspaces to a path that doesn't contain a colon.
+     * This fixes an issue on Windows where saving map workspaces would always fail.
+     */
+    public static void migratePath(MinecraftServer server) {
+        var manager = server.getOverworld().getPersistentStateManager();
+
+        // Don't overwrite a migrated file, if one exists
+        var file = manager.getFile(MapWorkspaceManager.KEY);
+        if (file.isFile()) return;
+
+        var legacyFile = manager.getFile(MapWorkspaceManager.LEGACY_KEY);
```

Comment:
I suppose there’s a risk that if this ever gets replaced with nio, we’ll start seeing the crash here too - what about wrapping this also in the try/catch?